docs/AboutUs.md

```diff
@@ -31,9 +31,9 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 ### Johnny Doe
```

Comment:
Need to change the name to follow #21 except for the picture.

Reply:
yeah I've already updated the rest! :)